tools/NodeConfigCompare/configuration/cloud_formation_template.yaml

```diff
@@ -968,49 +909,76 @@ Resources:
       Timeout: 60
       Code:
         ZipFile: |
+          import urllib.request
+          from urllib.parse import urlparse
           import boto3
+          import logging
+          import json
+          import os
+          import glob
           import cfnresponse
-          import traceback
+
+          os.chdir('/tmp')
```

Comment:
I don't think this is required right?

Reply:
I added this because in line 944 its easy for running the glob command.
When I remove this I had to add the file.split() in 948

The glob (glob.glob(' /tmp/* ')) returns the whole path and creates this creates the whole path in the s3 bucket

`
# s3_client.meta.client.upload_file(file, Bucket, Key=f'{path}/{file.split("/")[-1]}')
`